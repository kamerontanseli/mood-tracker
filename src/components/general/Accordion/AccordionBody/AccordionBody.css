.AccordionBody {
  margin-top: 20px;
}