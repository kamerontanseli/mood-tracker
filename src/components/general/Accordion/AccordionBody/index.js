import "./AccordionBody.css";
import React from "react";

const AccordionBody = ({ children }) => (
  <div className="AccordionBody">{children}</div>
);

export default AccordionBody;
