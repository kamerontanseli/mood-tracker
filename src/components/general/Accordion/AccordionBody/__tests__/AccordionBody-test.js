import React from 'react'
import renderer from 'react-test-renderer'
import AccordionBody from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <AccordionBody>Hello world</AccordionBody>
    );
    expect(snapshot).toMatchSnapshot()
  })
})