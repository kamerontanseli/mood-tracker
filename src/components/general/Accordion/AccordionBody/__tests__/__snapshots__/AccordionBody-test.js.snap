// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<div
  className="AccordionBody"
>
  Hello world
</div>
`;
